ci/autoresearch/test_lpc_ws2812_loopback.py

```
@@ -0,0 +1,270 @@
+        "num_leds",
+        "exp_bytes",
+        "dec_bytes",
+        "matched",
+        "mismatched",
+        "edges",
+    )
+    parts = csv.split(",")
+    if len(parts) != len(fields):
+        return {}
+    try:
+        return {k: int(v) for k, v in zip(fields, parts)}
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)